src/main/java/gov/cabinetoffice/gapfindapiadmin/controllers/ApiKeyController.java

```diff
@@ -1,18 +1,22 @@
 package gov.cabinetoffice.gapfindapiadmin.controllers;
 
+import gov.cabinetoffice.gapfindapiadmin.service.ApiKeyService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.servlet.ModelAndView;
 
 @Controller
 @RequestMapping("/api-keys")
 @RequiredArgsConstructor
 public class ApiKeyController {
 
+    private final ApiKeyService apiKeyService;
+
     @GetMapping
-    public ModelAndView showKeys() {
-        return new ModelAndView("organisation-api-keys");
+    public String showKeys(Model model) {
+        model.addAttribute("apiKeys", apiKeyService.getApiKeysForFundingOrganisation(2));
```

Comment:
Can you add a TODO to take this param from the request. Means Sonar will warn us about it when we push code and it can be tracked and not forgotten about